recaf-ui/src/main/java/me/coley/recaf/ui/control/config/ConfigLanguage.java

```diff
@@ -37,7 +37,6 @@ public ConfigLanguage(ConfigContainer instance, Field field) {
 
 		selectionModel.selectedItemProperty().addListener((observable, oldValue, newValue) -> {
 			// Canceling (seen below) re-selects the old value.
-			// This checks for that case and skips showing a duplicate 'Are you sure?' prompt.
```

Comment:
Why was this comment removed?

Reply:
Because restarting is no longer necessary I presume

Reply:
> Because restarting is no longer necessary I presume

This.